index.html

```diff
@@ -6,10 +6,45 @@
     <link rel="stylesheet" href="css/index.css" />
     <title>Abdullahi Js Calculator</title>
   </head>
-  <body>
-    <main role="main">
-      <h1>Hello, World</h1>
+  <body class="theme1">
+    <main role="main" class="main-container d-flex">
+      <header class="header d-flex">
+        <h1 class="header-text">calc</h1>
+        <h2 class="header-txt">THEME</h2>
+        <div class="theme-container">
+          <div class="theme-input">
+            <div class="dot"></div>
+            <label class="theme-1-label" for="theme-1">1</label>
+
+            <label class="theme-2-label" for="theme-2">2</label>
+
+            <label class="theme-3-label" for="theme-3">3</label>
+          </div>
+        </div>
+      </header>
+      <div class="screen d-flex">
+        <div class="screen-input">399,981</div>
```

Comment:
I dont think `input` is he right word here, perhaps `display`?
I would recommend using an id attribute as well so it can be used to target the element with js

Reply:
I have changed the input to display sir